packages/ibm-products-community/README.md

```diff
@@ -1,3 +1,25 @@
-# IBM Products community
+# Carbon for IBM Products Community
 
-TODO
+## About
+
+The purpose of the Community package is to host experimental components and
```

Comment:
While true, I think it’s not _only_ to host experimental components but also to enable teams to share code that may not be officially supported or documented through the design system guild.